### PR TITLE
ocaml-system, ocaml-config: quote ocamlc path on Win32

### PR DESCRIPTION
**Pull request by madroach**
This is necessary because of Windows' very special quoting rules in [system](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/system-wsystem?view=vs-2019#remarks) and [cmd](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd#remarks).
Could someone with access to a Windows machine test this on Cygwin, please?

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
@dra27 Could you take a look at this PR?

File: packages/ocaml-config/ocaml-config.1/opam [hunk 1]
```diff
@@ -14,7 +14,7 @@ depends: [
 ]
 substs: "gen_ocaml_config.ml"
 extra-files: [
-  ["gen_ocaml_config.ml.in" "md5=71f88c92be191cf4a0547f3c6196ef00"]
-  ["ocaml-config.install" "md5=8e50c5e2517d3463b3aad649748cafd7"]
+  ["gen_ocaml_config.ml.in" "md5=efb951a1b7bb6040b99dcb59267e1b56"]
```
Comment (kit-ty-kate) [inline packages/ocaml-config/ocaml-config.1/opam]: The hash doesn't match apparently:
```
#=== ERROR while compiling ocaml-config.1 =====================================#
Bad hash for /home/opam/.opam/repo/default/packages/ocaml-config/ocaml-config.1/files/gen_ocaml_config.ml.in
```

Comment (madroach) [reply packages/ocaml-config/ocaml-config.1/opam]: Indeed. I accidentally changed the wrong checksum. Fixed and tested.

**Comment by dra27** [comment]
Partly for curiosity, rather than approval, what’s your setup that you’ve hit this (which opam, machine configuration, etc)

**Comment by madroach** [comment]
I build a test environment in AppVeyor and tried to use ocaml-system:
[commit](https://github.com/madroach/ocaml-appveyor/commit/d3bd1d153353c9a4e3c13e82e3561cc5de1c6565)
[AppVeyor](https://ci.appveyor.com/project/madroach/ocaml-appveyor/builds/28484754)

**Comment by kit-ty-kate** [comment]
@dra27 should this be merged?

**Pull request closed without merging**